packages/notification-center/src/i18n/languages/da.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const EN: ITranslationEntry = {
```

Comment:
```suggestion
export const DA: ITranslationEntry = {
```

I think this was one was missed here 🙏